validator/accounts/prompt/prompt.go

```diff
@@ -110,20 +111,30 @@ func InputRemoteKeymanagerConfig(cliCtx *cli.Context) (*remote.KeymanagerOpts, e
 			return nil, err
 		}
 	}
-	crtPath, err := fileutil.ExpandPath(strings.TrimRight(crt, "\r\n"))
-	if err != nil {
-		return nil, errors.Wrapf(err, "could not determine absolute path for %s", crt)
+
+	crtPath, keyPath, caPath := "", "", ""
+	if crt != "" {
```

Comment:
When TLS is disabled, then passing cert paths is optional, and we should validate a path if it's provided.